test/Dockerfiles/bionic-py3.Dockerfile

```diff
@@ -3,11 +3,8 @@ SHELL ["/bin/bash", "-c"]
 
 # dependencies
 RUN apt-get update
-RUN apt-get install -y build-essential
 RUN apt-get install -y \
-    automake pkg-config libtool libgmp-dev
-RUN apt-get install -y \
-    python3-dev python3-pip python-virtualenv python3-pyqt4 python3-sip
```

Comment:
We added `pyqt4` and `sip` here to check that the GUI gets installed correctly when these are present on the system.
Maybe worth to keep this functionality somehow?

Reply:
There's a `requirements/gui.txt` file that has all the dependencies for the GUI and as a result, this is no longer needed.

Reply:
Understood.  Thanks.